mathesar/api/serializers/constraints.py

```diff
@@ -147,6 +148,10 @@ def create(self, validated_data):
         return serializer.create(validated_data)
 
     def run_validation(self, data):
+        if referent_table := data.get('referent_table', None):
```

Comment:
It would be better to use the `field` parameter of the `InvalidTableName` Exception class to specify which field of the body is failing the validation instead of having a different Exception.